demo/stable-diffusion/requirements.txt

```diff
@@ -1,6 +1,68 @@
-diffusers
-transformers
-nvidia-ml-py3
-ftfy
-torch
-Pillow
+accelerate==1.3.0
```

Comment:
We shouldn't need to pin all of the dependencies -- can you just pin the versions of `diffusers` and `torch`? That should be sufficient to pin all of their dependencies